tgui/packages/tgui/interfaces/Manufacturer/components/BlueprintButton.tsx

```diff
@@ -105,10 +110,10 @@ export const BlueprintButton = (props:BlueprintButtonProps) => {
           <LabeledList.Item
```

Comment:
Another "Mordent TODO", specifically re: the line above but I can't comment on that in the PR.